src/components/measurements/measurementsD3.js

```diff
@@ -27,14 +27,17 @@ export const layout = {
   diamondSize: 25,
   standardDeviationStroke: 2,
   overallMeanColor: "#000",
-  yAxisTickSize: 6
+  yAxisTickSize: 6,
+  yAxisColorByLineHeight: 7,
```

Comment:
I found the current line widths too subtle, but I understand you've picked them to mirror the SD lines. I personally preferred things a bit bigger  👇 but I haven't tested things on all resolutions. Not blocking, just a comment.

```js
  diamondSize: 50,
  standardDeviationStroke: 3,
  overallMeanColor: "#000",
  yAxisTickSize: 6,
  yAxisColorByLineHeight: 9,
  yAxisColorByLineStrokeWidth: 4
```

Reply:
Updated in c120508.

Things can look a little crammed when there are too many color groupings, but I do think it's an overall improvement.